vizro-core/docs/pages/user-guides/assets.md

```diff
@@ -264,12 +264,12 @@ It's essential to understand the relationship between the targeted CSS class or
     [CardCSS]: ../../assets/user_guides/assets/css_change_card.png
 
 
-??? note "Order of serving CSS files"
+## Order of serving CSS files
 
-    CSS properties will be applied with the last served file taking precedence. The order of serving is:
+CSS properties will be applied with the last served file taking precedence. The order of serving is:
 
     1. Dash built-in stylesheets
     2. Vizro built-in stylesheets
     3. User assets folder stylesheets
```

Comment:
```suggestion
1. Dash built-in stylesheets
2. Vizro built-in stylesheets
3. User assets folder stylesheets
```

I think otherwise they are shown as code right now

Reply:
![Uploading Screenshot 2024-07-08 at 15.20.56.png…]()